### PR TITLE
Requests for Comment

### PR DESCRIPTION
**Pull request by brownsarahm**
Creates an initial outline for #101

### CODE REVIEW

**Review by arinbasu — APPROVED** [review]

File: policy/request_for_comment.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Requests for Comments
+
+The trainers leadersip will use an RFC process for actions related to:
```
Comment (marklcrowe) [inline policy/request_for_comment.md:3]:
```suggestion
The trainers leadership will use an RFC process for actions related to:
```

File: policy/request_for_comment.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- Propose changes to Trainer Certification Renewal process
+- Establishing Task Forces and Subcommittees within Trainers group
+- Changing to Instructor checkout requirements
+- Other topics as deemed appropriate
+  
+When this occurs, they will:
+  1. Create a public PR, issue, or discussion
+  1. Notify the trainer community via mailing list and slack
```
Comment (marklcrowe) [inline policy/request_for_comment.md:11]:
```suggestion
  1. Notify the trainer community via mailing list and Slack
```

**Review by marklcrowe — APPROVED** [review]

**Comment by marklcrowe** [comment]
Hi Sarah, I’ve made two minor typo corrections and approved this one.

Cheers

Mark

--
Mark Crowe
Skills Development Manager

[signature_2541019231]<https://www.qcif.edu.au/services/skills-development/>



Queensland Cyber Infrastructure Foundation Ltd (QCIF)<http://qcif.edu.au/>
***@***.******@***.***> | 0434 331588
www.qcif.edu.au<http://www.qcif.edu.au> | [signature_3779554332]  @qcifltd<https://twitter.com/qcifltd> @marklcrowe<https://twitter.com/marklcrowe> | [signature_824281633]  @qcif<https://www.linkedin.com/company/1664945>

From: Sarah Brown ***@***.***>
Date: Friday, 6 May 2022 at 06:38
To: carpentries/trainers ***@***.***>
Cc: Mark Crowe ***@***.***>, Review requested ***@***.***>
Subject: Re: [carpentries/trainers] Requests for Comment (PR #153)

@brownsarahm<https://github.com/brownsarahm> requested your review on: #153<https://github.com/carpentries/trainers/pull/153> Requests for Comment.

**Comment by brownsarahm** [comment]
The panel approved creating such a policy with #101, I think we can asynchronously approve this text and merge if unanimous or put on the agenda if anyone would prefer to discuss.

**Review by paocorrales — APPROVED** [review]

**Review by jcoliver — APPROVED** [review]

**Review by bonnyad — APPROVED** [review]

**Pull request merged**